etl/steps/data/garden/who/2024-03-24/self_inflicted_injuries.meta.yml

```diff
@@ -0,0 +1,54 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Mental Health
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+  title: WHO Mortality Database - Self-inflicted injuries
+
+tables:
+  self_inflicted_injuries:
+    variables:
+
+      death_rate_per_100_000_population:
+        title: Death rate from self-inflicted injuries per 100,000 population
+        unit: deaths per 100,000 population
+        short_unit: ''
+        description_short: Reported annual death rate from self-inflicted injuries per 100,000 people, based on the [underlying cause](#dod:underlying-cause-of-death) listed on death certificates.
+        description_from_producer: Underlying causes of death grouped under this heading are those grouped under the ICD codes X60-X84, Y870.
+        display:
+          numDecimalPlaces: 2
```

Comment:
Nit pick, I think 1 d.p. is probably enough here. I think we've moved towards having that as the default (even though that is not the case in grapher!), it just makes things a little less visually noisy.